starter-core/src/test/groovy/io/micronaut/starter/feature/kotlin/KtorSpec.groovy

```diff
@@ -73,50 +76,24 @@ class KtorSpec extends ApplicationContextSpec implements CommandOutputFixture {
     }
 
     @Unroll
-    void 'dependency is included with maven and feature ktor for language=#language'(Language language) {
+    void 'dependency is included with #buildTool and feature ktor for language=#language'(Language language, BuildTool buildTool) {
         when:
-        String template = new BuildBuilder(beanContext, BuildTool.MAVEN)
+        String template = new BuildBuilder(beanContext, buildTool)
                 .language(language)
                 .features(['ktor'])
                 .render()
 
+        BuildTestVerifier verifier = BuildTestUtil.verifier(buildTool, template)
+
         then:
-        template.contains("""
-    <dependency>
-      <groupId>io.micronaut.kotlin</groupId>
-      <artifactId>micronaut-ktor</artifactId>
-      <scope>compile</scope>
-    </dependency>
-""")
-        and:
-        template.contains("""
-    <dependency>
-      <groupId>io.ktor</groupId>
-      <artifactId>ktor-server-netty</artifactId>
-      <scope>compile</scope>
-    </dependency>
-""")
-        and:
-        template.contains("""
-    <dependency>
-      <groupId>io.ktor</groupId>
-      <artifactId>ktor-jackson</artifactId>
-      <scope>compile</scope>
-    </dependency>
-""")
-        and:
-        template.contains("""
-    <dependency>
-      <groupId>io.micronaut.kotlin</groupId>
-      <artifactId>micronaut-kotlin-runtime</artifactId>
-      <scope>compile</scope>
-    </dependency>
-""")
-        and:
-        !template.contains('<artifactId>micronaut-http-server-netty</artifactId>')
+        verifier.hasDependency("io.micronaut.kotlin", "micronaut-ktor", Scope.COMPILE)
+        verifier.hasDependency("io.ktor", "ktor-server-netty", Scope.COMPILE)
+        verifier.hasDependency("io.ktor", "ktor-serialization-jackson", Scope.COMPILE)
```

Comment:
Gotta love people renaming modules ;-)

Reply:
I think we should make those maven coordinates managed in Kotlin.